main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "3.40.0"
+    }
+  }
+  backend "s3" {
+    bucket = "terraformstateinfrarepository"
+    key    = "infraRepository/state"
+    region = "sa-east-1"
+    dynamodb_table = "terraformstatelocking"
```

Comment:
The values on lines 9 and 12 can have underline (_) betweens word? Would be easier to read. Or camel case, if you prefer.

Reply:
s3 don't allow _, so I change with -

Reply:
Looks good to me!